modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,200 +7,180 @@
 ----
-$ {orch-cli} get -o json pod __<workspace pod name>__  -n __<workspace namespace>__ | \
-    jq -r '.spec.containers[] | select(.name | startswith("theia-ide")).name'
+$ {orch-cli} exec -t deploy/{prod-id-short} --namespace={prod-namespace} -- \
+    keytool -list -keystore {prod-home}/cacerts \
+    | grep --after-context=1 custom-ca-certificates.pem
 ----
 
-* Look for a `che-trusted-ca-certs` `ConfigMap` object inside the workspace namespace:
+. Start a workspace, get the {orch-namespace} name in which it has been created: __<workspace_namespace>__, and wait for the workspace to be started.
+
+. Verify that the `che-trusted-ca-certs` config map contains your custom CA certificates.
+The command returns your custom CA certificates in PEM format:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
-$ {orch-cli} get cm che-trusted-ca-certs -n __<workspace namespace>__
+$ {orch-cli} get configmap che-trusted-ca-certs \
+    --namespace=__<workspace_namespace>__ \
+    --output='jsonpath={.data.custom-ca-certificates\.custom-ca-certificates\.pem}'
 ----
 
-* Check that the entries in the `che-trusted-ca-certs` `ConfigMap` object contain all the additional entries you added before. In addition, it can contain `ca-bundle.crt` reserved entry. View the entries:
+. Verify that the workspace pod mounts the `che-trusted-ca-certs` config map:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
-$ {orch-cli} get cm che-trusted-ca-certs -n __<workspace namespace>__ -o json | jq -r '.data | keys[]'
-ca-bundle.crt
-source-config-map-name.data-key.crt
+$ {orch-cli} get pod \
+    --namespace=__<workspace_namespace>__ \
+    --selector='controller.devfile.io/devworkspace_name=__<workspace_name>__' \
+    --output='jsonpath={.items[0:].spec.volumes[0:].configMap.name}' \
+    | grep che-trusted-ca-certs
 ----
 
-* Confirm that the `che-trusted-ca-certs` `ConfigMap` object is added as a volume in the workspace Pod:
+. Verify that the `universal-developer-image` container (or the container defined in the workspace devfile) mounts the `che-trusted-ca-certs` volume:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
-$ {orch-cli} get -o json pod __<workspace pod name>__ -n __<workspace namespace>__ | \
-    jq '.spec.volumes[] | select(.configMap.name == "che-trusted-ca-certs")'
-{
-  "configMap": {
-    "defaultMode": 420,
-    "name": "ca-certs"
-  },
-  "name": "che-self-signed-certs"
-}
+$ {orch-cli} get pod \
+    --namespace=__<workspace_namespace>__ \
+    --selector='controller.devfile.io/devworkspace_name=__<workspace_name>__' \
+    --output='jsonpath={.items[0:].spec.containers[0:]}' \
+    | jq 'select (.volumeMounts[].name == "che-trusted-ca-certs") | .name'
 ----
 
-* Confirm that the volume is mounted into containers, especially in the Che-Theia IDE container:
+. Get __<workspace_pod_name>__, the workspace pod name:
```

Comment:
I would retrieving the workspace pod name earlier. Before running the commands to look for pod volumes and container volumemounts. The commands would be simplified.

Reply:
That was my intention at the beginning, but it's the only command that doesn't have a variant that uses only values that the Che Dashboard displays. 

I believe it makes sense to use these values as much as possible.

I would prefer to find such a variant for this last step.